scipy/special/tests/test_extending.py

```diff
@@ -11,6 +12,9 @@
 # essential per https://github.com/scipy/scipy/pull/20487#discussion_r1567057247
 @pytest.mark.skipif(IS_EDITABLE,
                     reason='Editable install cannot find .pxd headers.')
+@pytest.mark.skipif((platform.system() == 'Windows' and
+                     sysconfig.get_config_var('Py_GIL_DISABLED')),
```

Comment:
Here and elsewhere--this is a bit crude--I'm using it as a proxy for "build isolation has been disabled on Windows" rather than really intending for it to have anything to do with the GIL. Those two things just happen to overlap in our wheel builds because `pyproject.toml` doesn't have static identifier support for free-threaded mode detection.